packages/material-ui/src/Radio/Radio.js

```diff
@@ -65,7 +65,9 @@ const Radio = React.forwardRef(function Radio(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiRadio' });
   const {
     checked: checkedProp,
+    checkedIcon = defaultCheckedIcon,
     color = 'primary',
+    icon = defaultIcon,
```

Comment:
You need to update the JSDoc in the `.d.ts` file to fix " icon: Error: JSDoc @default annotation not found. Add `@default <RadioButtonIcon />` to the JSDoc of this prop.". 

To verify, run `yarn docs:api:build` locally.